docs/user-guide/command-line-interface.md

```diff
@@ -361,6 +362,16 @@ This option instructs ESLint to try to fix as many issues as possible. The fixes
 1. This option throws an error when code is piped to ESLint.
 1. This option has no effect on code that uses processors.
 
+If you want to fix code from stdin or otherwise want to get the fixes without actually writing them to the file, use the [`--fix-dry-run`](--fix-dry-run) option.
```

Comment:
I think this link is broken -- should it go to `#--fix-dry-run` instead of `--fix-dry-run`?